telemetry-proxy/telemetry_proxy.go

```diff
@@ -30,7 +31,17 @@ var upgrader = websocket.Upgrader{
 	},
 }
 
+// Create a channel to send Protobuf telemetry messages to the WebSocket handler
+var telemetryChannel = make(chan *pb.Telemetry)
+
+// Map to store active WebSocket connections
+var connections = make(map[*websocket.Conn]bool)
```

Comment:
It seems like the map is never set to false. It is either inserted with the value true or removed from the map. Can this be an array instead?

Reply:
Arrays in Go are a fixed size I believe, so it's better to use a Map. It also makes looking up by connection easier (we don't have to keep track of indices)